lms/templates/navigation.html

```diff
@@ -49,7 +49,7 @@ <h1 class="logo">
 
     % if course:
     <h2 class="course-header"><span class="provider">${course.display_org_with_default | h}:</span>
-      ${course.display_number_with_default | h}
+      <span class="course-number">${course.display_number_with_default | h}</span>
```

Comment:
Similarly at L60 in this file, a `span class="course-name"` around the course name element